### DISCUSSION TITLE
Adversarial attacks on semantic segmentation

### DISCUSSION THREAD

**Discussion by saivineethabvns** [Trusted-AI/adversarial-robustness-toolbox#1302] [Q&A] [answered]
Hello,
I wanted to apply the adversarial attacks and defence methods to semantic segmentation models using this tool box without writing the code from scratch. Could you suggest how this can be extended. Is there any example notebook for the same?
Thanks in advance.

**Comment by beat-buesser** [comment, marked as answer]
Hi @saivineetha1998 Thank you for your interest in ART! Semantic segmentation is something we would like to support explicitly in the very near future, but right now you would have to implement an estimator for segmentation models following ART's APIs similar to the class `PyTorchObjectDetector` in #1295.
